src/consumer/__tests__/batch.spec.js

```diff
@@ -3,9 +3,27 @@ const Batch = require('../batch')
 describe('Consumer > Batch', () => {
   const topic = 'topic-name'
 
+  it('discards messages with a lower offset than the requested', () => {
+    const fetchedOffset = 3
```

Comment:
Not that it matters for these cases, but shouldn't `fetchedOffset` always be a string?

Reply:
Everything will be converted into a long so, technically yes, but in the end it doesn't matter that much. Numbers highlight better in the test